web/component/ads/view.jsx

```diff
@@ -50,16 +51,9 @@ function removeIfExists(querySelector) {
 }
 
 function Ads(props: Props) {
-  const {
-    location: { pathname },
```

Comment:
Not a blocker
If `location` is no longer used, removed it here, and also remove `withRouter` down below.